src/modules/endpoint/types.js

```diff
@@ -78,3 +78,70 @@ export type JsonEventType = {
   event: JsonEventInfoType,
   location: JsonLocationType
 }
+
+export type JsonExtraPostType = {
+  [key: string]: string
+}
+
+export type JsonExtraType = {
+  alias: string,
+  name: string,
+  url: string,
+  thumbnail: string,
+  post: ?JsonExtraPostType
+}
+
+export type JsonSprungbrettJobType = {
+  title: string, zip: string, city: string, url: string, employment: string, apprenticeship: string
+}
+
+// Generated with: https://transform.now.sh/json-to-flow-types/
+type AccommodationType = {
+  ofRooms: string[],
+  title: string,
+  location: string,
+  totalArea: number,
+  totalRooms: number,
+  moveInDate: string,
+  ofRoomsDiff: string[]
+}
+
+type CostsType = {
+  ofRunningServices: string[],
+  ofAdditionalServices: string[],
+  baseRent: number,
+  runningCosts: number,
+  hotWaterInHeatingCosts: boolean,
+  additionalCosts: number,
+  ofRunningServicesDiff: string[],
+  ofAdditionalServicesDiff: string[]
+}
+
+type LandlordType = {
+  firstName: string,
+  lastName: string,
+  phone: string
+}
+
+type FormDataType = {
+  landlord: LandlordType,
+  accommodation: AccommodationType,
+  costs: CostsType
+}
+
+export type OfferType = {
+  email: string,
+  formData: FormDataType,
+  createdDate: string
+}
+
+export type JsonLanguageType = { code: string, native_name: string }
+
+export type JsonCityType = {
+  name: string,
+  path: string,
+  live: boolean,
+  events: boolean,
+  extras: boolean,
+  name_without_prefix: string
+}
```

Comment:
not a fan of this massive types file

Reply:
I think this is oke because this file should be generated automatically anyway. Or the cms team should provide this to us.

Reply:
Imo also okay.